providers/sharedtest/components/externalservice/setup.ftl

```diff
@@ -1,10 +1,10 @@
 [#ftl]
 
-[#macro shared_externalservice_default_deployment_generationcontract occurrence ]
```

Comment:
I don't think we want to rename this file here, as it will leave the `externalservice` component without a setup routine

Reply:
I moved it from the shared provider into the sharedtest prvovider in 81a7046. The externalservice component hasn't had a setup routine before and doesn't really need one. 

The shared provider looks after the state and the sharedtest provider looks after the setup for testing purposes

Reply:
Gotcha. No worries then :)